### DISCUSSION TITLE
How to set UTF8-PTBR

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#1942] [General]
Hi guys!!

I recently discover and implement the Tactical RMM. I'm trying to translate pages editing files .js.

I'm having problemas with encode, becouse my language (Portuguese) have special characters (á à ç ã ô â, etc.).

When I change something (like Workstation to Estações de Trabalho), strangers characters appear in the word "Estações".

I tried changing locales in the Debian, charset in pages and other things, but not have sucess.

I need help to solve this. See sample bellow:
![image](https://github.com/user-attachments/assets/1c05c919-a38f-457c-8871-cc8e011b1c96)

Sorry, but I don't speak fluently English.

**Comment by dinger1986** [comment]
please look at this issue here for translation https://github.com/amidaware/tacticalrmm/issues/39

Probably changing this line https://github.com/amidaware/tacticalrmm/blob/411cbdffeeca75a6f475f22388e3036bbefafac1/api/tacticalrmm/tacticalrmm/settings.py#L97 to true will help and restart the rmm service

**Reply by ederpardeiro** [reply]
Thank you for your answer, but don't work for me. I change for "True" and restart system, but UTF8 Charset PT-BR don't work, unfortunately.

**Reply by dinger1986** [reply]
ok, thought it was worth a shot

**Comment by wh1te909** [comment]
what's the output of the following command: `sudo -iu postgres psql --list`

**Reply by ederpardeiro** [reply]
Hi!
I can see this:

![image](https://github.com/user-attachments/assets/575946a0-4660-485e-a401-09de5fbae5c3)

**Reply by wh1te909** [reply]
thanks, never mind. misread your screenshot. translation support isn't implemented yet so editing the files directly isn't going to work. we'll need to properly add translation support in the framework before you can start doing translations.

**Reply by ederpardeiro** [reply]
Thank you very much for your answer.

Ok... I will use in English until the implementation of translation support is ready.
